### PR TITLE
Fix README to reflect proper command line flag

### PR DESCRIPTION
**Pull request by alallier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Configs matching a given schema are sourced from the following locations in the
 - Command line argument set in the schema via `commandLineArg`.
 - Environment variable set in the schema via `envVar`.
 - Deployment config file declared via:
-  - Command line argument: `--deploy-config` or `--dc`.
+  - Command line argument: `--deploy-config` or `--df`.
```
Comment (lannonbr) [inline README.md]: I'd say either do

1) `--deploy-config` and `--dc`
2) `--deploy-file` and `--df`

I'd stick with one style or the other, not a mixture of the two. As well make sure it follows in `getDeployConfig.js`

Comment (alallier) [reply README.md]: My vote is 2 and this will help us keep an existing API in apps that will now use this

Comment (kethinov) [reply README.md]:
```suggestion
  - Command line argument: `--deploy-config-file` or `--df`.
```

Comment (alallier) [reply README.md]: I believe this will need a code change as well

Comment (kethinov) [reply README.md]: Yes, but if we're gonna go with `--df` it needs to match the longer version. Otherwise `--dc` makes more sense.

Comment (alallier) [reply README.md]: I like your suggestion of `--deploy-config-file` or `--df`.

@lannonbr what are your thoughts

Comment (lannonbr) [reply README.md]: I'm good with that

Comment (alallier) [reply README.md]: Alright thanks I'll make the changes

File: getDeployConfig.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const fs = require('fs')
 const os = require('os')
 const projectRoot = require('app-root-path')
 const appPackage = require(projectRoot.path + '/package.json')
-const commandLineArguments = require('yargs-parser')(process.argv.slice(2), { alias: { deployFile: ['df'] } })
+const commandLineArguments = require('yargs-parser')(process.argv.slice(2), { alias: { deployConfigFile: [ 'df'] } })
```
Comment (lannonbr) [inline getDeployConfig.js]:
```suggestion
const commandLineArguments = require('yargs-parser')(process.argv.slice(2), { alias: { deployConfigFile: ['df'] } })
```

Don't think we need this extra space. Commit it if you agree

Comment (alallier) [reply getDeployConfig.js]: This commit suggestion thing is awesome

**Comment by alallier** [comment]
@lannonbr let me squash that late commit, just to make it a little nicer

**Pull request merged**